kustomize/commands/internal/util/util_test.go

```diff
@@ -71,34 +60,30 @@ func TestGlobPatternsWithLoaderRemoteFile(t *testing.T) {
 	}
 
 	// test load remote file
-	resources, err := GlobPatternsWithLoader(fSys, ldr, []string{httpPath})
-	if err != nil {
-		t.Fatalf("unexpected load error: %v", err)
-	}
-	if len(resources) != 1 || resources[0] != httpPath {
-		t.Fatalf("incorrect resources: %v", resources)
-	}
+	resources, err := GlobPatternsWithLoader(fSys, ldr, []string{httpPath}, false)
+	require.NoError(t, err, "unexpected load error")
+	require.Equal(t, 1, len(resources), "incorrect resources")
+	require.Equal(t, httpPath, resources[0], "incorrect resources")
 
 	// test load local and remote file
-	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{httpPath, "/test.yml"})
-	if err != nil {
-		t.Fatalf("unexpected load error: %v", err)
-	}
-	if len(resources) != 2 || resources[0] != httpPath || resources[1] != "/test.yml" {
-		t.Fatalf("incorrect resources: %v", resources)
-	}
+	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{httpPath, "/test.yml"}, false)
+	require.NoError(t, err, "unexpected load error")
+	require.Equal(t, 2, len(resources), "incorrect resources")
+	require.Equal(t, httpPath, resources[0], "incorrect resources")
+	require.Equal(t, "/test.yml", resources[1], "incorrect resources")
 
 	// test load invalid file
 	invalidURL := "http://invalid"
-	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{invalidURL})
-	if err == nil {
-		t.Fatalf("expected error but did not receive one")
-	} else if err.Error() != invalidURL+" has no match: "+invalidURL+" not exist" {
-		t.Fatalf("unexpected load error: %v", err)
-	}
-	if len(resources) > 0 {
-		t.Fatalf("incorrect resources: %v", resources)
-	}
+	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{invalidURL}, false)
+	require.Error(t, err, "expected error but did not receive one")
+	require.Equal(t, invalidURL+" has no match: "+invalidURL+" not exist", err.Error(), "unexpected load error")
+	require.Equal(t, 0, len(resources), "incorrect resources")
+
+	// test load unreachable remote file with skipped verification
+	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{invalidURL}, true)
+	require.NoError(t, err, "unexpected load error")
+	require.Equal(t, 1, len(resources), "incorrect resources")
+	require.Equal(t, invalidURL, resources[0], "incorrect resources")
```

Comment:
Thanks for fixing up these tests and adding one for this feature.

I think it might be nice to rewrite these tests using test structs and subtests, like for the [remove tests here](https://github.com/kubernetes-sigs/kustomize/blob/master/kustomize/commands/edit/remove/removebuildmetadata_test.go) so that it is easier to read and distinguish each test. You don't need to do this as part of this PR, but it would be a welcome change if you'd like to submit a follow-up PR to do so.

Reply:
Yes, I was also thinking about that when I refactored the tests. Let me tackle that in a separate PR.